flow/server/flow_server.go

```diff
@@ -233,16 +233,16 @@ func (s *FlowServer) handleFlows(flows []*flow.Flow) {
 	}
 }
 
-func (s *FlowServer) handleStatus(status *flow.Status) {
-	s.subscriberEndpoint.SendStatus(status)
+func (s *FlowServer) handleStatus(stats *flow.Stats) {
+	s.subscriberEndpoint.SendStatus(stats)
```

Comment:
should be `handleStats` and `SendStats`

Reply:
indeed, I renamed it, forgot to rename this one